Sources/GRPC/TLSVerificationHandler.swift

```diff
@@ -85,14 +85,12 @@ internal class TLSVerificationHandler: ChannelInboundHandler, RemovableChannelHa
         return
     }
 
-    self.logger.debug("TLS handshake completed, negotiated protocol: \(String(describing: negotiatedProtocol))")
     if let proto = negotiatedProtocol, GRPCApplicationProtocolIdentifier(rawValue: proto) != nil {
```

Comment:
Can you remove the check against `GRPCApplicationProtocolIdentifier` since we know that the protocol we agreed upon must be one that we suggested in the first place? 

There's an edge case here (which we didn't handle correctly before) wherein the user could override the suggested protocols to include one that isn't in `GRPCApplicationProtocolIdentifier`, if that was the case now we'd log "unknown protocol" which isn't quite true.

Reply:
Ok!